src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -152,7 +156,7 @@ public void platformMotionDownAABBTest()
      * Test getNextPosition in a terrible box of terribleness.
      */
     @Test
-    public void getNextPositionTestRetricted()
+    public void getNextPositionTestRestricted()
```

Comment:
Ah, I did not notice that. Apparently no one else did in the other pull requests either. I'll fix it.